gatsby-browser.js

```diff
@@ -116,6 +117,9 @@ export const onInitialClientRender = () => {
             if (!isProduction()) LocalStore.set('i18n', 'ach')
         }
 
+        if (window.location.pathname.includes(besquare_signup_url))
+            window.location.href = careers_url
+
```

Comment:
If `besquare_signup_url` is an external url then how this redirect will take effect? Aren't we suppose to handle it in the button who has this link and change it to `careers_url` instead?

Reply:
True, I was told by Habib to add this to handle external redirection. As for the button it is completely removed so can't really test it by clicking it. Need to somehow redirect back to careers page if the users find this URL(**besquare_signup_url** ) on any social sharing website

Reply:
The internal url will be inside the check, and we’ll redirect to external url